README.md

```diff
@@ -44,59 +50,53 @@ cd generative_data_prep
+The JSON Lines format can be used for fine-tuning, or pre-training/continual pre-training. Each line in the `.jsonl` format should be a json object with a `prompt`, and `completion` element. For example:
 
-The output HDF5 files each contain two datasets:
-- \"input_ids\": sequences of tokens ids
-- \"token_type_ids\": describe the type of each token. The default id assignments are:
-  - id=0 for tokens in the prompt
-  - id=1 for tokens in the completion
-  - id=2 for \<eos\> tokens that serve as padding tokens (will not be trained to predict)
-  - id=3 for \<eos\> tokens at the end of articles, that define the attention boundary when training with article attention
+```
+{"prompt": "What did the fox do?", "completion": "The quick brown fox jumped over the lazy dog."}
+{"prompt": "How much wood does a woodchuck chuck?", "completion": "A woodchuck chucks 1000 wood."}
+{"prompt": "Who sells seashells by the sea shore?", "completion": "She sells seashells by the sea shore."}
+```
 
-### Holdout Evaluation Data
-To evaluate on a holdout set of data during training, pipeline.py can create splits of holdout evaluation data.
+If the JSON objects in your `.jsonl` contain keywords other than **prompt** and **completion**, refer to the `prompt_keyword` and `completion_keyword` flags [below](#prompt_keyword)
 
-To do this, include flags from only one of the two options below, only use one option or the other. Please review the [Flags](#flags) section for in detail descriptions of these flags.
-- To specify the number of training splits and evaluation splits directly, use the three flags `--num_training_splits=...`, `--num_dev_splits=...` and `--num_test_splits=...`
-- To specify the percentage of the data heldout for evaluation, you can specify `--dev_ratio=0.1` and `--test_ratio=...`, where 0.1 means that approximately 10% of the data will be included in the evaluation splits. You can also specify the `--num_training_splits=...` flag to control the total number of training splits, but we recommend to let this default.
+### `.txt` Format
 
-All this evaluation data will saved under the `output_path`, if you want to run evaluation on the eval_splits during training you must enable `do_eval` on SambaStudio.
+This format should be used for pre-training/continual pretraining, but not fine-tuning. With text files, all sequences are used as completions, so all processed sequences end up having empty prompts in the prompt/completion pair. For example:
```

Comment:
It looks like the example you provided here is a jsonline txt files are just text separated by newlines. Check out this example to understand 

https://github.com/sambanova/generative_data_prep/tree/enh-struct-reorg/tests/examples/pretraining_txt